package.json

```diff
@@ -19,24 +19,25 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
-    "discord.js": "^12.5.1",
-    "node-fetch": "^2.6.1",
+    "cluster": "^0.7.7",
```

Comment:
cluster is a build in module, the one on npm is 10 years old and is not something we would wanna use